entities/Product.java

```diff
@@ -1,13 +1,12 @@
 package entities;
 
+import java.util.Date;
 import java.util.List;
 
-public class Product {
```

Comment:
Do this refactor in another `PR` . Avoid working with different entities in the same `PR`